README.md

```diff
@@ -190,6 +194,29 @@ If you just want to experiment with Jenkins X, you can set force_destroy to true
 
 :warning: **Note**: If you set `force_destroy` to false, and run a `terraform destroy`, it will fail. In that case empty the s3 buckets from the aws s3 console, and re run `terraform destroy`.
 
+### EKS node groups
+This module provisions self-managed worker nodes by default.
+In order to provision EKS node groups (EKS managed worker groups) create a _main.tf_ with the following content:
+```terraform
+module "eks-jx" {
+  source  = "jenkins-x/eks-jx/aws"
+  enable_node_group   = true
+  enable_worker_group = false
+}
+
+output "vault_user_id" {
+  value       = module.eks-jx.vault_user_id
+  description = "The Vault IAM user id"
+}
+
+output "vault_user_secret" {
+  value       = module.eks-jx.vault_user_secret
+  description = "The Vault IAM user secret"
+}
+
+```
+:warning: **Note**: If you forget to set `enable_worker_group = false`, then the module will provision both self managed worker groups and node groups.
```

Comment:
So is the combination  `enable_node_group = true` and  `enable_worker_group = true` ever useful? Or should we internally when `enable_node_group = true` is set, set `enable_worker_group = false`?

Reply:
That is something I am not sure about. Since AWS allows it (both multiple worker groups and multiple nodegroups as well as a combination of both), should we not allow it? Why put an artificial restriction? Again all the choices with AWS ...

Reply:
I see. So, in that case, you would get worker groups and node groups at the same time. They would happily work together to form the cluster. Ok then.